policy_test.go

```diff
@@ -95,15 +97,132 @@ func validateLockNVIndex(t *testing.T, tpm *tpm2.TPMContext) {
 	}
 }
 
-func TestEnsureLockNVIndex(t *testing.T) {
-	tpm, _ := openTPMSimulatorForTesting(t)
+func TestIncrementDynamicPolicyCounter(t *testing.T) {
+	tpm := openTPMForTesting(t)
+	defer closeTPM(t, tpm)
+
+	key, err := rsa.GenerateKey(rand.Reader, 2048)
```

Comment:
Would shorter test keys work here too? e.g. 768 or something? Just curious.

Reply:
Some of these are loaded in to the TPM, and whilst the reference implementation supports key sizes in multiples of 1024 bits, only 2048 bits is a mandatory requirement of the PC Client Profile spec: https://trustedcomputinggroup.org/wp-content/uploads/PC-Client-Specific-Platform-TPM-Profile-for-TPM-2p0-v1p04_r0p37_pub.pdf ("Support for 2048-bit keys is required; the TPM SHOULD NOT support 1024-bit keys").

It might be worthwhile using smaller sizes in tests where it isn't loaded in to the TPM.

Reply:
I was going to switch to smaller keys for tests where they aren't loaded in to the TPM, but go's crypto library complains if I use smaller keys in crypto operations too. I've changed some tests to use smaller keys though.